Crash looping controller manager - panic: index out of range in cronjob controller

Happened in 1.5.2. 
In controller manager log:

```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:52
/usr/local/go/src/runtime/asm_amd64.s:2086
panic: runtime error: index out of range [recovered]
        panic: runtime error: index out of range

goroutine 474 [running]:
panic(0x22b7560, 0xc420010070)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
k8s.io/kubernetes/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/runtime/runtime.go:56 +0x126
panic(0x22b7560, 0xc420010070)
        /usr/local/go/src/runtime/panic.go:458 +0x243
k8s.io/kubernetes/vendor/github.com/robfig/cron.ParseStandard(0x0, 0x0, 0x394f960, 0xc400000002, 0xc420a42cc0, 0x38ced73)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/robfig/cron/parser.go:20 +0x4e6
k8s.io/kubernetes/pkg/controller/cronjob.getRecentUnmetScheduleTimes(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/cronjob/utils.go:123 +0x6c
k8s.io/kubernetes/pkg/controller/cronjob.SyncOne(0x0, 0x0, 0x0, 0x0, 0xc42039cde6, 0x7, 0x0, 0x0, 0xc42039ce00, 0x7, ...)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/cronjob/controller.go:162 +0x3ff
k8s.io/kubernetes/pkg/controller/cronjob.(*CronJobController).SyncAll(0xc420c1c780)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/cronjob/controller.go:120 +0x611
k8s.io/kubernetes/pkg/controller/cronjob.(*CronJobController).SyncAll-fm()
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/cronjob/controller.go:93 +0x2a
k8s.io/kubernetes/pkg/util/wait.JitterUntil.func1(0xc4208ff090)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:96 +0x5e
k8s.io/kubernetes/pkg/util/wait.JitterUntil(0xc4208ff090, 0x2540be400, 0x0, 0xc420b6f101, 0xc420068180)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:97 +0xad
k8s.io/kubernetes/pkg/util/wait.Until(0xc4208ff090, 0x2540be400, 0xc420068180)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/wait/wait.go:52 +0x4d
created by k8s.io/kubernetes/pkg/controller/cronjob.(*CronJobController).Run
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/controller/cronjob/controller.go:93 +0x105
```

@soltysh @kubernetes/sig-apps-bugs

--- a/Godeps/Godeps.json
+++ b/Godeps/Godeps.json
@@ -2215,8 +2215,8 @@
 		},
 		{
 			"ImportPath": "github.com/robfig/cron",
-			"Comment": "v1-40-g783cfcb",
-			"Rev": "783cfcb01fb00c48f740c9de79122bd410294149"
+			"Comment": "v1-53-gdf38d32",
+			"Rev": "df38d32658d8788cd446ba74db4bb5375c4b0cb3"
 		},
 		{
 			"ImportPath": "github.com/rubiojr/go-vhd/vhd",

